frontend/src/components/GoalCards/StandardGoalCard.js

```diff
@@ -146,7 +146,8 @@ export default function StandardGoalCard({
 
   const contextMenuLabel = `Actions for goal ${id}`;
   const menuItems = [];
-
+  // For monitoring goals, only admins can delete
+  const canPerformMonitoringGoalOperations = isMonitoringGoal ? isAdmin(user) : true;
```

Comment:
this name is kind of weird don't you think? For a non-monitoring goal, every user "canPerformMonitoringGoalOperations". Would rather see it as canDelete, canReopen, etc and do the check within there, even if it means more lines it'll be more readable in the future

Reply:
I have updated both to hopefully be a little clearer.